packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -394,14 +410,14 @@ const ruleset: any = {
     /// ARM RPC rules for Put patterns
     ///
 
-    // RPC Code: RPC-Put-V1-01
-    PathForPutOperation: {
-      description: "The path for 'put' operation must be under a subscription and resource group.",
+    // RPC Code: RPC-Put-V1-01, RPC-Get-V1-11
+    PathForResourceGroupOperation: {
+      description: "The path must be under a subscription and resource group.",
```

Comment:
To get this PR in, lets revert this change and address the comments I have as a separate PBI in a subsequent sprint,.

Reply:
pl remove these changes since they are not scoped to tracked resource currently, as they are incorrect. And lets make the proper fix in the next sprint for this rule.